OHHTTPStubs/Sources/Swift/OHHTTPStubsSwift.swift

```diff
@@ -158,6 +158,20 @@ public func isMethodDELETE() -> OHHTTPStubsTestBlock {
 }
 
 /**
+ * Matcher for testing an `NSURLRequest`'s **absolute url**.
+ *
+ * e.g. the absolute url is `https://api.example.com/signin` in `https://api.example.com/signin`
+ *
+ * - Parameter scheme: The absolute url to match
+ *
+ * - Returns: a matcher (OHHTTPStubsTestBlock) that succeeds only if the request
+ *            has the given absolute url
+ */
+public func isAbsoluteUrl(_ url: String) -> OHHTTPStubsTestBlock {
```

Comment:
The problem with `isAbsoluteUrl` is also that it doesn't follow the Swift naming convention, letting us believe that the expected parameter is an `URL` and not a `String`.
Otoh just "isAbsoluteString" without context (as it's a free function, not a method on a tour) is also ambiguous… maybe `isAbsoluteURLString`? But might be too long. Tbh I honestly don't know what's best suited (especially at 0:30am 😅)